Documentation/gettingstarted/hubble.rst

```diff
@@ -0,0 +1,143 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _hubble_gsg:
+
+*************************************************
+Networking and security observability with Hubble
+*************************************************
+
+This guide provides a walkthrough of setting up a local multi-node Kubernetes
+cluster on Docker using `kind <https://kind.sigs.k8s.io/>`_ in order to
+demonstrate some of Hubble's capabilities.
+
+If you haven't read the :ref:`intro` yet, we'd encourage you to do that first.
+
+The best way to get help if you get stuck is to ask a question on the `Cilium
+Slack channel <https://cilium.herokuapp.com>`_.  With Cilium contributors
+across the globe, there is almost always someone available to help.
+
+.. include:: kind-install.rst
+
+.. note::
+   The cluster nodes will remain in state ``NotReady`` until Cilium is deployed.
+   This behavior is expected.
+
+Deploy Cilium and Hubble
+========================
+
+.. include:: k8s-install-download-release.rst
+
+Pre-load images into the kind cluster so each node does not have to pull
+them:
+
+.. parsed-literal::
+
+  docker pull cilium/cilium:|IMAGE_TAG|
```

Comment:
It would make sense to have all of the images here - `cilium/cilium`, `cilium/cilium-operator-generic` `cilium/hubble-relay` and `cilium/startup-script`, but that is a long list - how much difference does this really make if no image were pre-loaded?

Reply:
These are not pulled by every node.

Reply:
It would actually be a waste of resources to pre-load images that are not deployed on every node.